src/components/core/prism.css

```
@@ -215,4 +215,6 @@ pre[class*="language-"] {
 
 .gatsby-highlight {
   max-width: 100%;
+  width: -webkit-fill-available;
```

```suggestion
  width: 100%;
```

I checked chrome, firefox and edge on windows this fixes the problem. When we can I would like to try to avoid browser specific fixes like this, unless you know that 100% won't work in some cases (I only checked the total chats bit)?

---

Changed in https://github.com/livechat/livechat-public-docs/pull/572/commits/ca01d621c7764627c6144e4a84cddfaef96f650b. Thanks for paying attention 👍. Seems to work on all browsers, without IE but...on IE doc generally has a problem with the operation.